api/v1alpha1/openapi.yaml

```diff
@@ -1865,19 +1865,23 @@ components:
         skipServerVerification:
           type: boolean
           description: 'Skip remote server verification'
-    GitHttpRepoSpec:
+        token:
+          type: string
+          description: 'The token for auth with HTTP transport'
+          format: password
+    HttpRepoSpec:
       type: object
       additionalProperties: false
       properties:
         repo:
```

Comment:
Wondering if we should rename `repo` as now it can be a generic URL, and we use `repository` to reference a repository name so it can be confusing to distinguish.